resources/assets/less/select2.less

```diff
@@ -74,10 +74,10 @@
 //Multiple select
 .select2-container--default {
   .select2-selection--multiple {
-    border: 1px solid @gray;
-    border-radius: @input-radius;
+    border: 1px solid @gray !important;
+    border-radius: 0 !important;
     &:focus {
-      border-color: @light-blue;
+      border-color: @light-blue !important;
```

Comment:
I'm by no means an expert in CSS but I think using `important` is not recommended. I edited the `public/css/dist/all.css` and put this in the end (in the end because I found several similar selectors before, and this way it's the last style to be applied):
```.select2-container--default .select2-selection--multiple,
.select2-selection .select2-selection--multiple {
  border: 1px solid #d2d6de;
  border-radius: 0;
  padding: 6px 12px;
  height: 34px;
}```

These are the same as the single select2 selection and therefore the input takes the same style as the others. Let me know if I made myself clear or not and if this helps fix this. Also, I don't know why applying the changes to `all.css` works and `select2.less` don't... but 🤷

Reply:
When you build the less files the `all.css` gets overwritten. Which was really throwing for a loop because everything else I edited won't reflect in the `all.css`. 
noted on the `!important` tag usage. I'll remove that.

Reply:
Ah that makes sense, thanks for your explanation about `all.css`, let me see if I can find something more then.

Reply:
Hey, big breakthrough!!, your changes aren't reflected because that file is not compiled when